hyperdome/client/tor_connection_dialog.py

```diff
@@ -155,15 +145,11 @@ def run(self):
                 self.canceled_connecting_to_tor.emit()
 
         except BundledTorCanceled:
-            self.common.log(
-                "TorConnectionThread", "run", "caught exception: BundledTorCanceled"
-            )
+            self.__log.warning("Caught exception: BundledTorCanceled")
             self.canceled_connecting_to_tor.emit()
 
         except Exception as e:
-            self.common.log(
-                "TorConnectionThread", "run", "caught exception: {}".format(e.args[0])
-            )
+            self.__log.exception("unknown exception connecting to tor")
```

Comment:
Shouldn't you add `exc_info=True` here?

Reply:
Logger.exception() is shorthand for Logger.error(exc_info=True) :)

Reply:
:O That's so useful